umake/frameworks/baseinstaller.py

```diff
@@ -396,7 +398,8 @@ def decompress_and_install_done(self, result):
             UI.return_main_screen(status_code=1)
 
         self.post_install()
-
+        if self.desktop_filename:
```

Comment:
I meant just for the condition, then we would still use desktop (or a property) to pass to add_exec_link

if self.exec_path:
            add_exec_link(self.exec_path, self.desktop_filename.split('.')[0])